MyToolsYourToolsClient/src/app/offers-list/offers-list.component.html

```diff
@@ -1,13 +1,14 @@
-<a [routerLink]="['/offer-view/', offer.id]" class="list-group-item list-group-item-action" *ngFor="let offer of offers">
-  <div class="row">
-    <div class="col-sm-2">
-      <img class="my-image-size" src="{{offer.imgSrc}}">
+<a [routerLink]="['/offer-view/', offer.id]" class="list-group-item list-group-item-action"
+    *ngFor="let offer of offers">
+    <div class="row">
+        <div class="col-sm-2">
+            <img class="offer-image" src="{{offer.imgSrc}}">
+        </div>
+        <div class="col-sm-4">
+            <span class="offer-name">{{offer.name}}</span>
+        </div>
+        <div class="col-sm-6">
+            <span class="offer-info">{{offer.name}}, {{searchGroupName(offer.groupId)}}</span>
```

Comment:
wlasnie czemu drugi raz offer.name ? Ja u siebie w pr juz zrobilem tak ze wyswietla sie kategoria zamiast tego name drugi raz.

Reply:
Ktoś tak kiedyś zrobił i tak zostało, jutro rano postaram się to zmienić :)